client/src/components/uploads/UploadUserImage.js

```diff
@@ -1,31 +1,23 @@
-import React, { useEffect, useState } from "react"
+import React, { useState } from "react"
 import Dropzone from "react-dropzone"
 
 const UploadUserImage = ({ userPayload, setUserPayload }) =>{
-	const [newUploadFormData, setNewUploadFormData] = useState({
-		image: {}
-	})
 
-	const handleImageUpload = (acceptedImage)=>{
-		setNewUploadFormData({
-			...newUploadFormData,
-			image: acceptedImage[0]
-		})
-	}
+	const [source, setSource] = useState("")
 
-	const addUpload = (event)=>{
-		event.preventDefault()
+	const handleImageUpload = (acceptedImage) => {
+		setSource(URL.createObjectURL(acceptedImage[0]))
 		setUserPayload({
 			...userPayload,
-			image: newUploadFormData.image
+			image: acceptedImage[0]
 		})
 	}
 
 	return (
 		<>
 			<h5>Profile Image Uploads</h5>
 
-			<form className="load primary" onSubmit={addUpload}>
+			<form className="load primary">
```

Comment:
if this component is rendered within the registration form, I don't think an additional `form` tag is needed here since it will be a part of the registration form